tools/setup-dev.sh

```diff
@@ -351,26 +353,23 @@ git submodule update --init || exit 1
 [[ ! -e "src/gravel/cephadm/cephadm.bin" ]] && \
   ln -fs ../ceph.git/src/cephadm/cephadm src/gravel/cephadm/cephadm.bin
 
-if [ -d venv ] ; then
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+if [ -d $SCRIPT_DIR/venv ] ; then
     echo
     echo "Detected an existing virtual environment:"
-    echo "  > $(realpath venv)"
+    echo "  > $(realpath $SCRIPT_DIR/venv)"
     if yes_no "Blow it away"; then
-        rm -rf venv || exit $?
+        rm -rf $SCRIPT_DIR/venv || exit $?
     fi
     echo
 fi
 
+# Set up Virtual Env for dev tools
+$PYTHON -m venv $SCRIPT_DIR/venv || exit 1
 
-# We need system site packages because librados python bindings are only
-# available as a package, they're not on pypi (and Tim thinks they probably
-# never will be). Setting --system-site-packages here means the venv will
-# work properly if someone uses it inside the image, where we need to be
-# able to fall back to system packages to get librados.
-$PYTHON -m venv --system-site-packages venv || exit 1
-
-source venv/bin/activate
-pip install -r src/requirements.txt -U || exit 1
+source $SCRIPT_DIR/venv/bin/activate
+pip install -r $SCRIPT_DIR/requirements.txt -U || exit 1
```

Comment:
@jhesketh though the PR is merged, I don't understand why do you want this?????
Once get rid off --system-site-packages for tools/venv I now it will rebuild libvirt-python instead of using the system one if we put it to requirements.txt.